connector/src/main/java/org/geysermc/connector/network/LoggingPacketHandler.java

```diff
@@ -40,15 +43,17 @@
 public class LoggingPacketHandler implements BedrockPacketHandler {
 
     protected GeyserConnector connector;
-    protected GeyserSession session;
+    // Sessions have a client id number from 0-n and are looked up in this list by client id
+    @Getter
+    protected ArrayList<GeyserSession> sessions = new ArrayList<GeyserSession>(1);
```

Comment:
Just a minor nitpick, we generally specify collections like this with the interface itself (List in this example) and then initialize it with the actual class being used. Also, `new ArrayList<GeyserSession>` should be replaced with `new ArrayList<>` as `GeyserSession` is already inferred.

So this should be `protected List<GeyserSession> sessions = new ArrayList<>(1);`

Reply:
Done.